login/login.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package login
```

Comment:
Let's have a package-level doc comment explaining what this package
is for, please.